### ISSUE TITLE
ParseError PHP7

### ISSUE THREAD

**Issue by j-combee** [maximebf/php-debugbar#264]
In php7 they introduced a new error class, ParseError http://php.net/manual/en/class.parseerror.php.
This error type conflicts with the ExceptionsCollector::addException(Exception $e);

I have fixed this myself with adding a instanceof check like this:

```
    /**
     * Adds an exception to be profiled in the debug bar
     *
     * @param Exception $e
     */
    public function addException(Exception $e)
    {
        $this->exceptions[] = $e;
        $previousError = $e->getPrevious();
        if ($previousError instanceof Exception) {
            if ($this->chainExceptions && $previous = $e->getPrevious()) {
                $this->addException($previous);
            }
        }
    }
```

I put this in a issue and not in a push request since i do not know if there are more new classes that should be considered or if this fix is a correct fix for this issue.

**Comment by rossbearman** [comment]
I've just run into this issue myself. As all exceptions and errors in PHP 7 inherit from the Throwable class, I changed the signatures of `addException(Exception $e)` and `formatExceptionData(Exception $e)` to `addException(Throwable $e)` and `formatExceptionData(Throwable $e)`, which works correctly in the scenarios I've tested but obviously isn't particularly clean as the code refers to Exceptions everywhere.

**Comment by jamiehd** [comment]
Any update on this? Is there any plans to support php7 or should this project be considered abandoned? If I work on a PR will it be accepted?

Thanks

**Comment by marceux** [comment]
@jamiehd I think pull requests are monitored and the code is updated occasionally; please make the change because I'd need it, hahaha

**Comment by barryvdh** [comment]
Yes, I think you could probably remove the type hint and just add phpdocs, right?
I will accept PR's :)

**Comment by JCMais** [comment]
Any plans for a release with support for PHP 7?

**Comment by DevNIX** [comment]
Following the conversation in #279:

I feel like adding an error with the `addException` method, as the message will appear in the "Exceptions" tab on the debug bar, besides the `addThrowable` should be the best idea.

Maybe removing the PHP typehint and checking the instance in runtime to maintain compatibility with PHP 5 and support the `Throwable` interface if defined would be a good idea?
